backend/src/services/auth/auth-login-service.ts

```diff
@@ -212,6 +213,9 @@ export const authLoginServiceFactory = ({
     });
     // send multi factor auth token if they it enabled
     if (userEnc.isMfaEnabled && userEnc.email) {
+      const user = await userDAL.findById(userEnc.userId);
```

Comment:
I think you missed when a locked user is trying to do login. It should fail even before reaching the mfa state

Reply:
this one actually already errors out for the user upon clicking "Continue with Email"
<img width="1499" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/90506288-3a59-4bfe-bce0-dbaf13219aeb">